packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -461,14 +470,16 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         //
         // Explicit checks:
         // - The game must not have already been initialized.
-        // - An output root cannot be proposed at or before the genesis block.
+        // - An output root that is more than 1 day old cannot be proposed.
 
         // INVARIANT: The game must not have already been initialized.
         if (initialized) revert AlreadyInitialized();
 
-        // Do not allow the game to be initialized if the root claim corresponds to a block at or before the
-        // configured genesis block number.
-        if (l2BlockNumber() <= GENESIS_BLOCK_NUMBER) revert UnexpectedRootClaim(rootClaim());
```

Comment:
There is no guarantee that genesis block number is more than a day ago. It may wind up being the hard fork we deploy fault proofs in and thus be basically "right now".